bottlecap/src/lifecycle/invocation/triggers/mod.rs

```diff
@@ -96,14 +96,32 @@ pub fn parameterize_api_resource(resource: String) -> String {
     result.join("/")
 }
 
+#[must_use]
+pub fn get_default_service_name(
+    instance_name: &str,
+    fallback: &str,
+    aws_service_representation_enabled: bool,
+) -> String {
+    if !aws_service_representation_enabled {
+        return fallback.to_string();
+    }
+
+    instance_name.to_string()
+}
+
```

Comment:
Does this have to be a method? Can't it just be inside of the `resolve_service_name`? Don't see the method being used anywhere else